### ISSUE TITLE
Toolbar buttons in CKEditor 5 do not meet WCAG minimum size requirements (24x24px)

### ISSUE THREAD

**Issue by marthinal** [ckeditor/ckeditor5#18215]
## 📝 Provide a description of the improvement

According to [WCAG 2.5.5 Target Size (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html), interactive elements such as buttons should have a minimum target size of 24 by 24 pixels to ensure accessibility for all users, particularly those using touch devices or with motor impairments.

When using CKEditor 5 (as integrated in Drupal 10), accessibility auditing tools such as Siteimprove flag toolbar buttons — especially split buttons like Numbered List or Bulleted List — as too small.

<img width="976" alt="Image" src="https://github.com/user-attachments/assets/ca8cdf18-2f92-4fa7-be75-07ba3ba0d686" />

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by Witoso** [comment]
@Comandeer can I ask you to take a look?

**Comment by Comandeer** [comment]
I can confirm that the split buttons are not wide enough. According to [the standard](https://www.w3.org/TR/WCAG22/#target-size-minimum), the target size should be at least 24px×24px or the spacing should be adjusted to meet the second criterion:

> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the [bounding box](https://www.w3.org/TR/WCAG22/#dfn-bounding-boxes) of each, the circles do not intersect another target or the circle for another undersized target;

**Comment by cory-skeers** [comment]
In our instance, (integrated into Drupal 10), we are overriding the style at
https://github.com/ckeditor/ckeditor5/blob/815c25ac80508e9799e02610a3b949d220fb5493/packages/ckeditor5-theme-lark/theme/ckeditor5-ui/components/dropdown/splitbutton.css#L35
to set the `min-width: 24px` rather than unsetting the width entirely.

It's a slight visual difference for a textless button, roughly a 5px increase in width, but seems to fit the intention of the accessibility guidance well.
